lib/engine/game/g_18_zoo/ability/increase_distance_for_train.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module Engine
```

Comment:
why is this in 18zoo abilities specificially?

Reply:
I need to associate a train to the ability, so I added an ability here; should I use just “description” and keep the data inside game?